slack/resource_conversation.go

```diff
@@ -228,7 +228,7 @@ func findExistingChannel(ctx context.Context, client *slack.Client, name string,
 		}
 	}
 	// looked through entire list, but didn't find matching name
-	return nil, fmt.Errorf("name_taken, but could not find channel")
+	return nil, fmt.Errorf("could not find channel with name %s", name)
```

Comment:
this error message was sort of specific to the channel adoption use case. made more general.